src/d/alexa-openwebif/source/intents/togglemute.d

```diff
@@ -20,9 +22,18 @@ final class IntentToggleMute : BaseIntent
 	///
 	override AlexaResult onIntent(AlexaEvent, AlexaContext)
 	{
-		immutable res = apiClient.vol("mute");
-
 		AlexaResult result;
+		Vol res;
+		try
+		{
+			res = apiClient.vol("mute");
+		}
+		catch (Exception e)
+		{
+			result = returnError(this);
```

Comment:
why first set result if you return it anyway ? lets make this a oneliner (applies to all other intents aswell)